.github/workflows/key4hep.yml

```diff
@@ -85,3 +84,36 @@ jobs:
                   --inputFiles=bbudsc_3evt_SIM.edm4hep.root \
                   --outputFileBase=bbudsc_3evt_GaudiRec \
                   --detectorModel=${{ matrix.detector_model }}
+
+  # Reconstruction with the v11 model (FCCee) using EDM4hep inputs and outputs
+  gaudi-v11-reco:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        detector_model: [ILD_l5_v11]
+        key4hep_build: [sw.hsf.org, sw-nightlies.hsf.org]
```

Comment:
Since we are currently not expecting this to work with the release we could also remove this from this matrix here.